sklearn/linear_model/cd_fast.pyx

```diff
@@ -205,8 +209,11 @@ def enet_coordinate_descent(floating[::1] w,
 
     with nogil:
         # R = y - np.dot(X, w)
-        for i in range(n_samples):
-            R[i] = y[i] - dot(n_features, &X[i, 0], n_samples, &w[0], 1)
+        copy(n_samples, &y[0], 1, &R[0], 1)
```

Comment:
Is it not possible to copy 2 memoryviews without using BLAS functions?

Reply:
Had some issues getting that to pass on CI with some older versions of Cython. Also saw other functions in this file were using BLAS to do the copy. So figured this was preferred. Though we can look into this more closely if you prefer.